proto/wechaty/puppet/message.proto

```diff
@@ -150,15 +153,15 @@ message MessageSendTextResponse {
 
 message MessageSendMiniProgramRequest {
   string conversation_id = 1;
-  string mini_program = 2;
```

Comment:
We must never re-use a protocol buffer id, such as change number `2` from `string` to `MiniProgramPayload`.

Keep the old value, and use a new value instead.

```
string mini_program = 2;
MiniProgramPayload mini_program = 3;
```